.github/workflows/tests.yml

```diff
@@ -78,3 +78,8 @@ jobs:
       - name: Run tests
         run: |
           pytest tests/ --deselect=tests/notebooks -n auto
+
+      - name: Run codecov
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        run: |
+          bash <(curl -s https://codecov.io/bash) -t 7ca04b7c-a541-485e-ad98-8adc99f6700f -X gcov
```

Comment:
Q: Shouldn't we change this?
(I think this is your personal token, right?) we need one for the org